.angular-cli.json

```diff
@@ -18,7 +18,7 @@
       "tsconfig": "tsconfig.app.json",
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
-      "styles": [],
+      "styles": ["./assets/css/bootstrap.min.css"],
```

Comment:
The aim of angular-custom-modal library was to avoid having to use Bootstrap to make modals work. I'm not at all against showing that bootstrap does really work with this library, but we should probably have a toggle functionality for that on the demo page (a sort of "Inject Boostrap").

Reply:
what about putting bootstrap class on example (with styleUrl), not as a project css? This way on the example will work with bootstrap min, but as lib, the user will be free to use any bootstrap implementation.